packages/WaffleGrid/WaffleGrid.jsx

```diff
@@ -0,0 +1,145 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import Box from '@tds/core-box'
+import Link from '@tds/core-link'
+import Image from '@tds/core-image'
+import Paragraph from '@tds/core-paragraph'
+import { colorGreyGainsboro } from '@tds/core-colours'
+
+import { safeRest } from '@tds/util-helpers'
+
+const StyledLogoRow = styled(Box)({
```

Comment:
Is it necessary to style Box, which is already a TDS component. Could you use a regular div instead of styling Box, and add the necessary styles to it.